mlir/test/Dialect/Vector/vector-transfer-flatten.mlir

```diff
@@ -501,8 +505,8 @@ func.func @transfer_write_non_contiguous_src(
 /// TODO: Move to a dedicated file - there's no "flattening" in the following tests
 ///----------------------------------------------------------------------------------------
 
-func.func @fold_unit_dim_add_basic(%arg0 : vector<1x8xi32>) -> vector<1x8xi32> {
-   %add = arith.addi %arg0, %arg0 : vector<1x8xi32>
+func.func @fold_unit_dim_add_basic(%vec_0 : vector<1x8xi32>) -> vector<1x8xi32> {
+   %add = arith.addi %vec_0, %vec_0 : vector<1x8xi32>
    return %add : vector<1x8xi32>
 }
 // CHECK-LABEL:   func.func @fold_unit_dim_add_basic(
```

Comment:
I suppose getting rid of  %[[VAL_X:.*]]: system will be as part of 
`// TODO: Align naming and format with e.g. vector-transfer-permutation-lowering.mlir` ?

Reply:
We may need to accept a tiny bit of inconsistency here ... 😅 

IMHO, for small tests, there's not much harm when using `VAL_X` - it's still possible to follow what's going on. However, for larger tests, these variables make analysing tests super hard.